### ISSUE TITLE
Error when testing with Page Speed Insight

### ISSUE THREAD

**Issue by jaynakus** [quentin-sommer/react-useragent#1]
When testing page in https://developers.google.com/speed/pagespeed/insights

Site gets error with react-useragent
```
TypeError undefined is not a function
webpack:///node_modules/@quentin-sommer/react-useragent/es/UserAgent.js:17
  | props = _objectWithoutProperties(_ref, ['children', 'returnfullParser']);
  |  
  | var validProps = Object.keys(props).filter(function (prop) {
  | return availableProps.includes(prop);
  | });
  | var ret = validProps.some(function (prop) {
  | return ua.uaResults[prop];

```

**Comment by quentin-sommer** [comment]
Hi,

I can't reproduce on my side using client-side `react-useragent` in a simple `create-react-app` application. Can you share with me a reproducible example of the bug or be more specific about your configuration? Are you using server-side rendering or any framework?

**Comment by jaynakus** [comment]
Hi,
https://quentin-sommer.github.io/react-useragent/

Tried your example site. On PageSpeed Insights it shows blank screen, as well. My site shows blank screen, too. I have bug tracking code deployed to site, so received above error to bugtracker bugsnag. Normally it is not reproducible.

**Comment by quentin-sommer** [comment]
Turns out page speed didn't recognize `Array.includes` as a function on a default ES6 array export. Fixed in `1.2.5`!
